.envrc

```diff
@@ -0,0 +1 @@
+use nix
```

Comment:
Do you think .envrc can be added to gitignore and not part of this commit ?

The reason being different people might want use to use different `.envrc`.

Reply:
Yes, that's reasonable.